docs/Search-auxiliary-files.md

```diff
@@ -0,0 +1,170 @@
+Storage for bookkeeping and performance reasons. These data files are called **auxiliary files**. The data files
+mentioned here are those are explicitly managed by the search subsystem. Other data files exist (manually created,
+created by the statistics subsystem, etc.). Those will not be covered here but are mentioned in the job-specific
+documentation that uses them as input.
+
+Each search auxiliary file is copied to the individual region that a [search service](../src/NuGet.Services.SearchService/README.md)
+is deployed. For nuget.org, we run search in four regions, so there are four copies of each of these files.
+
+The search auxiliary files are:
+
+  - [`downloads/downloads.v2.json`](#download-count-data) - total download count for every package version
+  - [`owners/owners.v2.json` and change history](#package-ownership-data) - owners for every package ID
+  - [`verified-packages/verified-packages.v1.json`](#verified-packages-data) - package IDs that are verified
+  - [`popularity-transfers/popularity-transfers.v1.json`](#popularity-transfer-data) - popularity transfers between package IDs
+
+## Download count data
+
+The `downloads/downloads.v2.json` file has the total download count for all package versions. The total download count
+for a package ID as a whole can be calculated simply by adding all version download counts.
+
+The downloads data file looks like this:
+
+```json
+{
+  "Newtonsoft.Json": {
+    "8.0.3": 10508321,
+    "9.0.1": 55801938
+  },
+  "NuGet.Versioning": {
+    "5.6.0-preview.3.6558": 988,
+    "5.6.0": 10224
+  }
+}
+```
+
+The package ID and version keys are not guaranteed to have the original (author-intended) casing and should be treated
+in a case insensitive manner. The version keys will always be normalized via standard `NuGetVersion` normalization rules
+(e.g. no build metadata will appear, not leading zeroes, etc.).
+
+If a package ID or version does not exist in the data file, this only indicates that there is no download count data and
+does not imply that the package ID or version does not exist on the package source. It is possible for package IDs or
+versions that do not exist (perhaps due to deletion) to exist in the data file. 
+
+The order of the IDs and versions in the file is undefined.
+
+This file has a "v2" in the file name because it is the second version of this data. The "v1" format is still produced
+by the statistics subsystem and has a less friendly data format.
+
+The class for reading and writing this file to Blob Storage is [`DownloadDataClient`](../src/NuGet.Services.AzureSearch/AuxiliaryFiles/DownloadDataClient.cs).
+
+## Package ownership data
+
+The `owners/owners.v2.json` file contains the owner information about all package IDs. While this file is being updated
+periodically, the set of package IDs that changed since the last iteration is written to a file with a path pattern like
```

Comment:
```suggestion
The `owners/owners.v2.json` file contains the owner information about all package IDs. While this file is updated
periodically, the set of package IDs that changed since the last iteration is written to a file with a path pattern like
```

The second sentence was a little confusing for me at first. What do you think of something like this?

```suggestion
The `owners/owners.v2.json` file contains the owner information about all package IDs. Each time this file is updated,
the set of package IDs that changed is written to a "change history" file with a path pattern like
```